vuepress/docs/v7.0/docs/reference/deployment-structure.md

```diff
@@ -4,154 +4,128 @@ sidebarDepth: 2
 
-#### Customization
-It is unlikely that the _component-manager_ will be customized as part of an Entando implementation.
-
 ## entando-k8s-service
-The _entando-k8s-service_ acts as an abstraction layer to fetch data from kubernetes APIs. The primary 
-functionality is in discovering and making available for installation Entando plugins. The 
-_entando-k8s-service_ is invoked by the _component-manager_. 
+The **entando-k8s-service** acts as an abstraction layer to fetch data from Kubernetes APIs. Its primary 
+function is to discover and make available Entando plugins for installation. The 
+**entando-k8s-service** is invoked by the ECM. 
 * GitHub: <https://github.com/entando-k8s/entando-k8s-service/>
 * DockerHub: <https://hub.docker.com/repository/docker/entando/entando-k8s-service/>
 
-#### Customization
-It is very unlikely that the _entando-k8s-service_ will be customized as part of an Entando implementation.
-
-## keycloak
-The _entando-keycloak_ project is an extension of the base Keycloak images. The extension provides the default 
-themes for Entando, a customized realm and clients, and adds the Oracle ojdbc jars for connection to Oracle 
-databases.
+## Keycloak
+The **entando-keycloak** project is an extension of the base Keycloak images. It provides default themes for Entando, customized realm and clients, and Oracle JDBC jars for connecting to Oracle databases.
 * GitHub: <https://github.com/entando/entando-keycloak/>
 * DockerHub: <https://hub.docker.com/repository/docker/entando/entando-keycloak/>
 
 #### Customization
-The keycloak image will often be customized as part of an Entando implementation. Common extensions will 
-include changing the theme, adding default connections, adding default social logins, adding default clients, 
-or other changes. 
+The Keycloak image can be customized as part of an Entando implementation. Some common extensions are: 
+* change the theme 
+* add default connections 
+* add default social logins 
+* add default clients 
 
 ## Other Key Repositories 
 ### entando-core
-The entando-core project is a J2EE application that exposes APIs for the Entando CMS, includes the legacy 
-admin console, and includes the portal-ui project that performs the server side composition for pages 
-rendered via an Entando application. Note that only the composition is performed server side. 
-Javascript code is rendered on the client. The entando-core is realized via an instance that includes the 
-WAR files generated from a core build as dependencies. In a default deployment this is the _entando-de-app_.
+The **entando-core** project is a J2EE application that exposes APIs for the Entando CMS, including the admin console and the portal UI project that performs the server side composition for pages rendered via an Entando application. Note that only the composition is performed server side. JavaScript code is rendered client-side. The **entando-core** is realized via an instance that includes the 
```

Comment:
Entando Application

split first sentence in 2 for clarity and ease of reading

"realized" is awkward

Reply:
As long as it made sense, I passed on doing a deep dive on this doc. Just wanted to get rid of the more blatant issues.